recipe/meta.yaml

```diff
@@ -27,20 +29,19 @@ requirements:
 test:
   requires:
     - python 2.7.*  # [win and py27]
-    - python 3.4.*  # [win and py34]
     - python 3.5.*  # [win and py35]
 
   commands:
     # Verify headers are present.
-    - test -d "${PREFIX}/include/jsoncpp"           # [unix]
+    - test -f "${PREFIX}/include/json/json.h"      # [unix]
+    - if not exist %PREFIX%\\Library\\include\\json\\json.h exit 1  # [win]
```

Comment:
Kind of a small thing, but we can use `%LIBRARY_INC%` to shorten this line a bit.